cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/RetryingSqlTransactionRunner.java

```diff
@@ -55,6 +60,11 @@ public RetryingSqlTransactionRunner(StructuredTableAdmin tableAdmin, DataSource
                                cConf.getBoolean(Constants.Metrics.STRUCTURED_TABLE_TIME_METRICS_ENABLED),
                                scanFetchSize);
     this.metricsCollectionService = metricsCollectionService;
+    this.maxRetries = cConf.getInt(Constants.Dataset.DATA_STORAGE_SQL_TRANSACTION_RUNNER_MAX_RETRIES, 20);
```

Comment:
Don't add defaults here. Defaults should come from cdap-default.xml

Reply:
ack